Add placeholders for current tardis location

Chew opened the pull request:
This adds 7 PlaceholderAPI placeholders pertaining to the current location of the TARDIS.

Benefits over just using /tardis find:
Lets you see more (biome, direction, etc)
Lets you separate out x,y,z,world for use on scoreboard, GUIs, etc.

Messages in order of the screenshot below
Placeholder `%tardis_current_location_x%`
Placeholder `%tardis_current_location_y%`
Placeholder `%tardis_current_location_z%`
Placeholder `%tardis_current_location_world%`
Placeholder `%tardis_current_location_biome%`
Placeholder `%tardis_current_location_direction%`
Placeholder `%tardis_current_location%`
Command `/tardis find`

![example](https://cdn.discordapp.com/attachments/457230849277165610/696850957430161448/unknown.png)

Code's awful but I managed to get it working so I'm happy!

eccentricdevotion commented:
Yep, that's pretty ugly, you should be using `ResultSetTardisID#fromUUID(String uuid)` if all you need is the TARDIS id - https://github.com/eccentricdevotion/TARDIS/blob/v4.0/src/main/java/me/eccentric_nz/TARDIS/database/ResultSetTardisID.java#L56

Chew commented:
ResultSetTardisID#fromUUID(String uuid) returns a boolean of if it exists or not, then I can do
if (exist) {
  int id = rsti.getTardis_Id()

and go on my merry way!

Fixed that everywhere, also moved some variable declarations outside of the switch/case

eccentricdevotion commented:
Cool, you can keep going though:
```
if (rsti.fromUUID(uuid)) {
   // do stuff
}
```
No need to create a boolean variable.

Chew commented:
Ah, peak efficiency! I'll commit that in a bit

Chew commented:
All good :+1:, I recommend a squash and merge lol
